docs/docs/building-and-running.md

```diff
@@ -81,9 +79,9 @@ helm install kube-registry-proxy -f deploy/helm/kube-registry-proxy.yml magda-io
 Now you can build the docker containers locally - this might take quite a while so get a cup of tea.
 
 ```bash
-eval $(minikube docker-env) # (If you haven't run this already)
-npx lerna run build --stream --concurrency=1 --include-dependencies # (if you haven't run this already)
-npx lerna run docker-build-local --stream --concurrency=1 --include-filtered-dependencies
+eval $(minikube docker-env) # (If you're running in minikube and haven't run this already)
+yarn lerna run build --stream --concurrency=1 --include-dependencies # (if you haven't run this already)
+yarn lerna run docker-build-local --stream --concurrency=1 --include-filtered-dependencies
```

Comment:
I think that's the newer lerna switch?
```suggestion
yarn lerna run docker-build-local --stream --concurrency=1 --include-dependencies
```

Reply:
@sajidanower23  I upgraded the lerna version in package so users should get `3.20.2` when they run `yarn install`

Reply:
Yep, and lerna 3.18 or above requires `--include-filtered-dependencies`, does it not?